parametric/spec/otel_trace.py

```diff
@@ -0,0 +1,67 @@
+import math
+from typing import List
+from typing import TypedDict
+
+
+class OtelSpan(TypedDict):
+    name: str
+
+
+class OtelSpanContext(TypedDict):
+    trace_id: bytes
+
+
+Trace = List[OtelSpan]
+
+
+def find_otel_span_in_traces(traces: List[Trace], span: OtelSpan) -> OtelSpan:
```

Comment:
Instead of using this method to find otel spans, can we set a unique tag on all otel spans generated by system tests or use a unique naming convention for otel spans. This check could become flaky and hard to maintain.

Reply:
I will remove this in favor of `span = get_span(test_agent)`